phpcs.xml.dist

```
@@ -9,29 +9,59 @@
     <file>./src</file>
     <file>./tests</file>
 
-    <rule ref="./vendor/endouble/symfony3-custom-coding-standard/Symfony3Custom">
+    <!-- Include the whole Symfony2 standard -->
+    <rule ref="vendor/escapestudios/symfony2-coding-standard/Symfony2">
         <!-- GENERAL -->
         <exclude name="Generic.Files.LineLength.MaxExceeded"/>
         <exclude name="PSR2.Methods.FunctionCallSignature.SpaceBeforeCloseBracket"/>
 
         <!-- FUNCTIONS -->
         <!-- Ignore missing function docblocks as we hint parameter and return types using proper typehints -->
-        <exclude name="Symfony3Custom.Commenting.FunctionComment.Missing"/>
-        <exclude name="Symfony3Custom.Commenting.FunctionComment.MissingReturn"/>
+        <exclude name="Symfony2.Commenting.FunctionComment.Missing"/>
+        <exclude name="Symfony2.Commenting.FunctionComment.MissingReturn"/>
         <!-- We don't care if parameter names all line up as if one changes then the diff will be noisy -->
-        <exclude name="Symfony3Custom.Commenting.FunctionComment.SpacingAfterParamType"/>
-        <exclude name="Symfony3Custom.Commenting.FunctionComment.SpacingAfterParamName"/>
-        <exclude name="Symfony3Custom.Commenting.FunctionComment.MissingParamTag"/>
+        <exclude name="Symfony2.Commenting.FunctionComment.SpacingAfterParamType"/>
+        <exclude name="Symfony2.Commenting.FunctionComment.SpacingAfterParamName"/>
+        <exclude name="Symfony2.Commenting.FunctionComment.MissingParamTag"/>
+        <exclude name="Symfony2.Commenting.FunctionComment.MissingParamComment"/>
```

Added because symfony2 set a severity level for this = 0.
By default phpcs hide errors/warning with serverity level less than 5. So this was excluded to display the error.
@see: https://github.com/djoos/Symfony-coding-standard/blob/master/Symfony/ruleset.xml#L75